wpimath/src/main/native/include/frc/controller/ElevatorFeedforward.h

```diff
@@ -54,55 +59,66 @@ class ElevatorFeedforward {
       this->kA = units::unit_t<ka_unit>{0};
       wpi::math::MathSharedStore::ReportWarning("kA defaulted to 0;");
     }
+    if (dt <= 0_ms) {
+      wpi::math::MathSharedStore::ReportError(
+          "period must be a positive number, got {}!", dt.value());
+      this->m_dt = 20_ms;
+      wpi::math::MathSharedStore::ReportWarning("period defaulted to 20 ms.");
+    }
   }
 
   /**
-   * Calculates the feedforward from the gains and setpoints.
+   * Calculates the feedforward from the gains and setpoints assuming continuous
+   * control.
    *
-   * @param velocity     The velocity setpoint, in distance per second.
-   * @param acceleration The acceleration setpoint, in distance per second².
+   * @param velocity     The velocity setpoint.
+   * @param acceleration The acceleration setpoint.
    * @return The computed feedforward, in volts.
+   * @deprecated Use the current/next velocity overload instead.
    */
-  constexpr units::volt_t Calculate(units::unit_t<Velocity> velocity,
-                                    units::unit_t<Acceleration> acceleration =
-                                        units::unit_t<Acceleration>(0)) {
+  [[deprecated("Use the current/next velocity overload instead.")]]
+  units::volt_t Calculate(units::unit_t<Velocity> velocity,
+                          units::unit_t<Acceleration> acceleration) const {
     return kS * wpi::sgn(velocity) + kG + kV * velocity + kA * acceleration;
   }
 
   /**
-   * Calculates the feedforward from the gains and setpoints.
+   * Calculates the feedforward from the gains and setpoints assuming continuous
+   * control.
    *
-   * @param currentVelocity The current velocity setpoint, in distance per
-   *                        second.
-   * @param nextVelocity    The next velocity setpoint, in distance per second.
+   * @param currentVelocity The current velocity setpoint.
+   * @param nextVelocity    The next velocity setpoint.
    * @param dt              Time between velocity setpoints in seconds.
    * @return The computed feedforward, in volts.
    */
+  [[deprecated("Use the current/next velocity overload instead.")]]
   units::volt_t Calculate(units::unit_t<Velocity> currentVelocity,
                           units::unit_t<Velocity> nextVelocity,
                           units::second_t dt) const {
     // Discretize the affine model.
-    //
-    //   dx/dt = Ax + Bu + c
-    //   dx/dt = Ax + B(u + B⁺c)
-    //   xₖ₊₁ = eᴬᵀxₖ + A⁻¹(eᴬᵀ - I)B(uₖ + B⁺cₖ)
-    //   xₖ₊₁ = A_d xₖ + B_d (uₖ + B⁺cₖ)
-    //   xₖ₊₁ = A_d xₖ + B_duₖ + B_d B⁺cₖ
-    //
+
+    //  dx/dt = Ax + Bu + c
```

Comment:
Is this covered by the elevator feedforward derivation? If so, please reference the derivation. If not, please revert the indentation changes here.

Reply:
It is.  I'll reference.  I figured since this was deprecated we would leave it until removal.